node-webkit/node-webkit.d.ts

```diff
@@ -219,3 +219,54 @@ declare module "nw.gui" {
     export var Shell: Shell;
 
 }
+
+interface Notification extends EventTarget {
+    /**
+     * The text displayed in the notification body
+     * @readonly
+     */
+    body:string;
+
+    /**
+     * The direction the popup will sho
+     * @readonly
+     */
+    dir:"auto"|"ltr"|"rtl";
+
+    /**
+     * The icon shown in the notification
+     * @readonly
+     */
+    icon:string;
+
+    /**
+     * The title shown in the notification
+     * @readonly
+     */
+    title:string;
+
+    /**
+     * The notification language
+     * @readonly
+     */
+    lang:string;
+
+    new( title:string, options:NotificationOptions ):Notification;
+    close():void;
+
+    onclick:() => void;
+    onshow:() => void;
+    onerror:( error:Error ) => void;
+    onclose:() => void;
+}
+
+declare var Notification: Notification;
+
```

Comment:
I think this needs to be declared as a class instead of interface + var 
```ts
declare class Notification extends EventTarget {
constructor( title:string, options:NotificationOptions );
```

This declaration permits you to do something like this, which isn't very useful:
```ts
Notification.body = "foo"; 
```